### DISCUSSION TITLE
fcitx5中input method wayland协议的支持

### DISCUSSION THREAD

**Discussion by asterwyx** [fcitx/fcitx5#894] [Q&A] [answered]
看到了fcitx5里面同时支持了input-method-unstable-v1和input-method-unstable-v2，并且代码中目前没有看到明显的互斥逻辑，fcitx5是支持这两种协议的热切换的吗？

**Comment by wengxt** [comment, marked as answer]
一般来说，Compositor 只支持一种，所以没有热切换的需求。
